Makefile.am

```diff
@@ -55,7 +56,7 @@ linux_platform_headers = \
 if HTOP_LINUX
 AM_LDFLAGS += -rdynamic
 myhtopplatsources = linux/Platform.c linux/IOPriorityPanel.c \
-linux/LinuxProcess.c linux/LinuxProcessList.c linux/LinuxCRT.c linux/Battery.c \
+linux/LinuxProcess.c linux/LinuxProcessList.c linux/LinuxCRT.c linux/Battery.c TemperatureMeter.c \
```

Comment:
This should probably go under `myhtopsources`

Reply:
Then it'll fail to build on FreeBSD and macOS.

Reply:
Nevermind, managed to add --enable-sensors.